docs/config/02-files.md

```diff
@@ -54,6 +54,11 @@ The `files` array determines which files are included in the browser, watched, a
 * **Default.** `false`
 * **Description.** Should the files be served from disk on each request by Karma's webserver?
 
+### `integrity`
+* **Type.** String
+* **Default.** `undefined`
+* **Description.** Set the `integrity` HTML attribute value to the `<script>` or the `<link>` tag to load the resource that matches the given pattern if the pattern is an absolute URL.
```

Comment:
> if the pattern is an absolute URL

QQ: Where is this assertion in the PR?

Reply:
Here: https://github.com/karma-runner/karma/pull/3797/files#diff-bbc76ee4392cb9602e2ce4d6081c2985bad45532441cec4da57b4a2d8a57b2b8R66

Reply:
@jginsburgn OK?

Reply:
@jginsburgn Can you approve?